### DISCUSSION TITLE
[ARCHIVED] Assignment 1 - Invisible Story

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#17] [Homework]
__Prompt__

Develop a simple app that documents the story of a culture that you feel is under-represented in public narratives. You can define culture and public narratives the way you want it. The app should convey some information to the reader, with at least text, images, and hyperlinks, and some interactivity.

__Skills__

- Develop familiarity with web development, HTML, CSS and JS.
- Get used to the UI and UX thinking
- Get familiar with manipulating and organizing digital collections

__Submission__:

- Upload a .zip file containing all of your HTML CSS, JS. (Using GitHub's upload, or a Google Drive link)
- Write a short summary of the project: what did you intend to do? How well do you consider you managed? What would you continue to work on, if you had more time?

**Comment by kf1735** [comment]
[Project-1.zip](https://github.com/periode/augmenting-gallery/files/11744924/Project-1.zip)

**Comment by chaerin-d** [comment]
My webpage "SSAM" is about the food culture in Korea. I tried to include a short description of SSAM and some examples of ingredients they can be but it is really based on personal taste. The meaning inside of SSAM also shows food etiquette in Korea, which can be really helpful, especially for children, who should have balanced nutrition, and not to act greedy of for their plates. I wanted to make people participate in SSAM by giving a GIF illustration that I created like a small game while they are dining in a group. 
I mostly set my color palette through CSS and I have about three different colors and font styles used. Then, I created 4 different pages for the HTML page and embedded links from the article and Youtube.

[file:///Users/chaerindo/Desktop/__/23-24%20NYU/23%20Fall/AG/SSAM.html](url)

* Video file for screen recording is available in Google Drive :)
* link to Google Drive [https://drive.google.com/drive/folders/1U61ryrCHaDF5_GAQjry0S5zThCcB5ipZ?usp=sharing](url)

**Reply by periode** [reply]
The overall design of the project is very nice, and gives a warm feeling of "handmade", especially with the drawn illustrations. The idea of sharing SSAM is also very well represented in the last page of the website, that is  interactive. Some recommendations for further design and development could be (1) make the menu more visible and explicit ("next page" does not mean much for the visitor"), and (2) use JavaScript to recreate the game without having to take a screenshot!

Grade: 98 (A)

**Comment by Caythian** [comment]
In this exercise, I made a website dedicated to origami or paper folding. I became personally interested in origami after I interviewed an origami artist in New York in my freshman year for a project. I like how rich the history of origami was, as well as how today there is still a group of enthusiasts who are devoted to making origami more popular and all-age friendly. I started with building a general website that contains the history of origami and some tutorials. However, I found the history section too typical. Therefore, I changed this part into “origami as a culture”, where the history and the contemporary development of origami are demonstrated from its ceremonial, recreational, and scientific aspects. The most challenging part of this website is to build an interactive menu bar. I tried hard to create that floating effect. One question that I discovered when making this website is how to simplify the codes. For example, I had to copy and paste the title, the menu, and the footer individually into each sub-page. I wonder if there’s a way to organize them as a template that I could use anytime I want without pasting all the codes. 
Here's the link to Google Drive: https://drive.google.com/drive/folders/1U78iQWP_oRJru9Dc0JTGMuG7ngCVOoOX?usp=sharing

**Reply by periode** [reply]
The research on origami is very well done and very well presented! In terms of content, you could put a little bit more description on the links in the virtual archive, so that the visitor does not have to open each link to know what it is about. Design-wise, the virtual archives are slightly off-centered, and the line-length is quite long, making it a bit tiring on the eyes, so maybe you could reduce the width of the content and center it using `margin: auto`. Finally, a suggestion to make your menu more cohesive with the theme would be to have two different icons: the closed menu could be an image of a folded origami, and the open menu could be an open of the same origami unfolded.

Grade: 98 (A)

**Comment by bhornik** [comment]
I created a website that defines the genre of “Samba Rock,” and gives a very broad overview of its cultural signifance in Brazil. Then I chose to get very specific by highliting the pioneers of the genre. I fell in love with samba rock and brazilian music this year, and I constantly find myself returning to the music of Tim Maia and Sandra de Sa (two artists highlited on the website). Yet, very few Americans know of, let alone listen to, Samba Rock, and therefore I feel like it deserves much more recognition. I began the process by mapping the entire website out on Adobe XD, then began to very slowly debug hundreds of silly errors until I had what the website is now.

.Zip and Video here: https://drive.google.com/drive/folders/1cUo1uUL9ify2j0xIgCiA1JyUGUUo_rYU?usp=sharing

**Reply by periode** [reply]
The website is very beautifully designed, very clear and pleasant to navigate, particularly the elements on the left giving a title to each page. However, it is not immediately clear what the website is about: about the history, or about featured artists? You could have grouped the artists under one section in the menu, and maybe historical/social context in another. Finally, it's very good to have links to youtube, but you can also embed the videos directly so that visitors do not have to leave your site.

Grade: 99 (A)

**Comment by lizalmonds** [comment]
My project is about snail mail and the origin of postal systems. It is a brief overview on the relevance of snail mail despite technological advancements in communication. I looked into the importance of postal systems and was intrigued on how it was integral for many major empires in the ancient world. If I had more time I would have included a timeline of how communication changed over time, and provide a brief overview on the tradition of love letters. I probably would have fleshed out the wire frame a bit more .  

Zip: https://drive.google.com/file/d/1DoD7RR3E0edxdgiKYhTjAno6sFnF5FW0/view?usp=drive_link 
Video: https://drive.google.com/file/d/1rneslWBZvdGHiXsqb2TH0vlnMaK7cRA9/view?usp=sharing

**Reply by periode** [reply]
Both the research and the design are very well done and very interesting, even though the research is a litte limited. For instance, you could have extended on differences/similarities between countries, representations in fiction, add further reading, mention the future of the postal system, etc. The use of the cursor and the animation on the postcards is also a nice coherent touch, which you could have pushed further by having different paper backgrounds in your historical overview (e.g. oldest paper, older paper, contemporary paper).

Grade: 98 (A)

**Comment by jacksonkkingg** [comment]
I created a site about the Roland Tr-909 drum machine, more specifically about what it means to music: reinvention. It moved the entirety of music into a more inventive headspace and led to the creation of numerous subgenres. It even popularized electronic dance music. I wanted my website to look very retro, keeping on theme with the drum machine, but I honestly think that wasn't enough. If I could add something, I would add buttons instead of hyperlinks on the page that takes you to the 909 samples. It would be much more interactive if someone could click on the buttons and the sounds would play, almost as if you are playing the machine.

ZIP: [https://drive.google.com/file/d/1XiBktbP285k8-HVxvuoBXB3TQhRM07QP/view?usp=sharing](url)

**Reply by periode** [reply]
The design is very clean and clear, and the research is well done. The videos really help to contextualise your points are well integrated. Some aspects of the website that could be developed further would be a better menu (clicking on the blue button gives us little clue as to where we are going). Finally, it's not super clear why you chose the featured media rather beyond just illustrations: maybe you could add more of them and add a legend?

Grade: 98 (A)

**Comment by alkashiwag** [comment]
I created a website focusing on the Portland Trail Blazers, an NBA team of my hometown. I feel the team is under-represented in the NBA because Portland is a small market team, there are much less residents and fans than other franchises such as the LA Lakers or NY Knicks. I think the website is able to get across important details and to offer insight to the disappointment of being a basketball fan in Portland. If I had more time to work on this project I would want to add more interactive elements with javascript and give every possible player mentioned their own page. I would also add more formatting as the text is sometimes distracting and weirdly spaced.
[assignment1.zip](https://github.com/periode/augmenting-gallery/files/12735489/assignment1.zip)

**Comment by zbyiii** [comment]
[Bingyi Zhang project 1.zip](https://github.com/periode/augmenting-gallery/files/12735665/Bingyi.Zhang.project.1.zip)

**Reply by periode** [reply]
The design is very simple and clear, and the content is very interesting, but the research is very limited (just one page on the whole website, which seems to be copy-pasted from Wikipedia), so it could be interesting to include a bit more, by looking at different sources, making comparisons with other practices to resist the patriarchy in other cultures, or just saying why it is meaningful to you. Finally, the lack of information on the Artworks section makes it complicated to understand what these are, so some clarification, maybe in the form of legends, would be very welcome.

Grade: 93 (A-)

**Comment by Ambraaaliu** [comment]
https://ambraaaliu.github.io/AugmentingGallery/InvisibleStories/

My web page is mainly about the introduction of guqin, starting with two stops related to guqin, the title is written in Xu Bing's "Book of heaven" , representing the current popular and classical combination of guqin. A Guqin video takes you to understand the sound of the guqin, and then introduces the structure of the guqin and its professional fingering rules.

**Reply by periode** [reply]
The website is very nicely laid out, and the background effect really sets the mood. In terms of content, however, it is a little bit limited: the explanatio that you gave during the presentation was very interesting, but whenever we look at this website, this context is lost and the pictures do not tell enough for non-mandarin speakers. To remediate, you could add legends, or split up the website into separate pages, where you explain in more details the construction, composition and fingering techniques.

Grade: 94 (A)

**Comment by RobbinHezhengWu** [comment]
My project is an attempt to tell the story of shamanism from the perspective of a practitioner. Distinguishing it from researches that have been criticized by cultural appropriation, this web page will provide a step-by-step introduction to what shamans believe about their cosmology, their source of power, and their oracles of shamanic practice. It also introduces the viewer to relevant artists and practicing groups.

google drive link: https://drive.google.com/drive/folders/10dr0M8H3Y6xK02w6cv-OzBbNVKVyNVxM?usp=drive_link

[Excersise1.zip](https://github.com/periode/augmenting-gallery/files/12735886/Excersise1.zip)

**Reply by periode** [reply]
The project is very well researched and very interesting to read, and provides perspectives from yesterday and today's practices of shamanism. The layout of the website is also very clear and easy to navigate. One point about that navigation is the use of anchors (jumping to IDs) that you have made: since it jumps automatically to the element with the id, it tends to cutoff the top of the element, so the trick is rather to jump to an element _before_ the one you want to navigate to. Finally, while the illustrations are very beautiful, it's not clear to me why they depict what they do: particularly, why is Van Gogh shown on the first image? The website does not mention if he was a shaman or not.

Grade: 98 (A)

**Comment by karldib** [comment]
This project is a website where i shared my culture, some of my experiences and meaningful moments I had in my life in Lebanon. I used some interaction as well as links to allow a pleasant user experience.

My zipped folder and screen recording: https://drive.google.com/drive/folders/13OOTecJfeCjpfQAj6ULUJZog384234Hp?usp=drive_link

**Reply by periode** [reply]
Your website gives a nice visual peek at your Lebanon, but it provides little to no context. A visitor would not know what this is about, why those pictures were chosen, or what the narrative is. Maybe you could include a section saying explicitly what this is about, who are the people and places in the pictures, why those moments are meaningful, etc. Additionally, there is very limited navigation (no way to go back to the main page), and some of the images do not display at all.

Grade: 92 (A-)

**Comment by liunikolai** [comment]
I am a big fan of industrial music. Through the Slovenian industrial band Laibach I discovered an entire fascinating movement of subversive art groups in Slovenia under the rule of Yugoslavia, which I would like to share with people. https://drive.google.com/drive/folders/1XnRO6tampBEnn_F7QarHZgXlYTHuDLY6?usp=drive_link

**Reply by periode** [reply]
The research is very well done, and the unique layout works well as a representation of the myriad of artists from this genre and era. It could have been nice, in the case of musicians, to embed directly a youtube video or some audio files to play the music as the visitor is browsing the website. Two aspects that could be improved: since your menu always opens a new tab, one can easily end up with 10 tabs open as they browse the content, and the home page has very little context, so adding a line on "avant-garde yugoslavia" (for example) could help frame the project for a visitor who has no idea about any of this.

Grade: 98 (A)
